docs/quick.rst

```diff
@@ -502,12 +503,6 @@ attribute. Note that a message is displayed to the screen when each
 bound is calculated, to indicate progress::
   
     >>> errors = gefit.est_errors()
-    gerr.fwhm lower bound:	-0.0326327
-    gerr.fwhm upper bound:	0.0332578
-    gerr.pos lower bound:	-0.0140981
-    gerr.pos upper bound:	0.0140981
-    gerr.ampl lower bound:	-0.0456119
-    gerr.ampl upper bound:	0.0456119
```

Comment:
I don't understand why these lines have been removed. They are written out and I'd want users to see this in the output. 

I think the issue is that this screen output is to the Sherpa logger and this is probably something that doctest doesn't handle well. Many years ago I did some work to catch the log output for some reason, but I forgot what that was for but it was docs related.

Reply:
I'll have to look why I did that, presumably because it's not printed to standard output. I might add a new handler to the logger to print all logger output to stdout, when run as a doctest. Alternatively, we can just skip this example from doctest (not just that one line, because then there is unexpected output, but simply not test that example). That's why it's good to convert just a few files and see where it goes and how we like it.

Reply:
The easiest solution is to show the output in a separate block that is not looked at by the doctest because it's not a block that looks like Python code. Slightly not ideal, but much easier and, I would argue, still good for this purpose. (Check tomorrow after I push to GH.)

Reply:
I've had a quick look and I can't see any obvious PRs where I added the grab-the-log-output-for-the-docs code, which probably means it's lost to the winds.